model/simulationtests/shadingcontrol.rb

```diff
@@ -0,0 +1,85 @@
+require 'openstudio'
+require 'lib/baseline_model'
+
+m = BaselineModel.new
+
+#make a 1 story, 100m X 50m, 1 zone core/perimeter building
+m.add_geometry({"length" => 100,
+                "width" => 50,
+                "num_floors" => 1,
+                "floor_to_floor_height" => 4,
+                "plenum_height" => 1,
+                "perimeter_zone_depth" => 0})
```

Comment:
I appreciate wanting to make it the smallest possible!

In this specific case, perhaps it's good idea here to test it out with at least 2 zones, and see what happens if a SC is shared across zones. What do you think @joseph-robertson?